data/sound-strings/pl.json/localize.json

```diff
@@ -30,17 +30,18 @@
 "in_2_5_kilometers":"Za dwa i pół kilometra",
 "in_3_kilometers":"Za trzy kilometry",
 "then":"Następnie",
-"take_the_1_exit":"Zjedź pierwszym zjazdem.",
-"take_the_2_exit":"Zjedź drugim zjazdem.",
-"take_the_3_exit":"Zjedź trzecim zjazdem.",
-"take_the_4_exit":"Zjedź czwartym zjazdem.",
-"take_the_5_exit":"Zjedź piątym zjazdem.",
-"take_the_6_exit":"Zjedź szóstym zjazdem.",
-"take_the_7_exit":"Zjedź siódmym zjazdem.",
-"take_the_8_exit":"Zjedź ósmym zjazdem.",
-"take_the_9_exit":"Zjedź dziewiątym zjazdem.",
-"take_the_10_exit":"Zjedź dziesiątym zjazdem.",
-"take_the_11_exit":"Zjedź jedenastym zjazdem.",
+"onto":"w",
```

Comment:
In my opinion it would be better to replace `"w"` with "na" because it sounds better with street names and in this case is universal.

Reply:
This is updated in the latest, thank you for your feedback! See below for discussion on sentence ordering as well.